packages/widget/src/App.tsx

```diff
@@ -39,17 +42,25 @@ export const App = forwardRef<WidgetDrawer, WidgetProps>(
 
 export const AppDefault = () => {
   const expandable = useExpandableVariant();
+
+  const { sdkConfig } = useWidgetConfig();
+
+  const isMultisigWalletEnabled = sdkConfig?.multisigConfig?.isMultisigSigner;
+
   return (
     <AppExpandedContainer>
-      <AppContainer>
-        <Header />
-        <FlexContainer disableGutters>
-          <AppRoutes />
-        </FlexContainer>
-        <PoweredBy />
-        <Initializer />
-      </AppContainer>
-      {expandable ? <RoutesExpanded /> : null}
+      {isMultisigWalletEnabled ? <MultisigWalletAlert /> : null}
+      <AppExpandedContainerMultisig>
+        <AppContainer>
+          <Header />
+          <FlexContainer disableGutters>
+            <AppRoutes />
+          </FlexContainer>
+          <PoweredBy />
+          <Initializer />
+        </AppContainer>
+        {expandable ? <RoutesExpanded /> : null}
+      </AppExpandedContainerMultisig>
```

Comment:
We discussed that this should be on Jumper, please move this component and all related stuff there.